src/bootstrap/config.rs

```diff
@@ -821,7 +821,14 @@ impl Config {
             .and_then(|output| if output.status.success() { Some(output) } else { None });
         if let Some(output) = output {
             let git_root = String::from_utf8(output.stdout).unwrap();
-            config.src = PathBuf::from(git_root.trim().to_owned())
+            let git_root = PathBuf::from(git_root.trim()).canonicalize().unwrap();
```

Comment:
```suggestion
            // We need to canonicalize this path to make sure it uses backslashes instead of forward slashes.
            let git_root = PathBuf::from(git_root.trim()).canonicalize().unwrap();
```